src/pks/flow/overland_pressure_pk.cc

```diff
@@ -69,60 +55,108 @@ OverlandPressureFlow::OverlandPressureFlow(Teuchos::ParameterList& pk_tree,
     iter_(0),
     iter_counter_time_(0.)
 {
-  if(!plist_->isParameter("conserved quantity key suffix"))
-    plist_->set("conserved quantity key suffix", "water_content");
-
   // set a default absolute tolerance
   if (!plist_->isParameter("absolute error tolerance"))
     plist_->set("absolute error tolerance", 0.01 * 55000.0); // h * nl
+
+  // get keys
+  conserved_key_ = Keys::readKey(*plist_, domain_, "conserved quantity", "water_content");
+
+  potential_key_ = Keys::readKey(*plist_, domain_, "potential", "pres_elev");
+  flux_key_ = Keys::readKey(*plist_, domain_, "mass flux", "mass_flux");
+  flux_dir_key_ = Keys::readKey(*plist_, domain_, "mass flux direction", "mass_flux_direction");
+  velocity_key_ = Keys::readKey(*plist_, domain_, "velocity", "velocity");
+  elev_key_ = Keys::readKey(*plist_, domain_, "elevation", "elevation");
+  pd_key_ = Keys::readKey(*plist_, domain_, "ponded depth", "ponded_depth");
+  pd_bar_key_ = Keys::readKey(*plist_, domain_, "ponded depth, negative",
+          Keys::getVarName(pd_key_)+"_bar");
+  wc_bar_key_ = Keys::readKey(*plist_, domain_, "conserved quantity, negative",
+          Keys::getVarName(conserved_key_)+"_bar");
+  cond_key_ = Keys::readKey(*plist_, domain_, "overland conductivity", "overland_conductivity");
+  uw_cond_key_ = Keys::readKey(*plist_, domain_, "upwind overland conductivity",
+          "upwind_overland_conductivity");
+  dens_key_ = Keys::readKey(*plist_, domain_, "molar density liquid", "molar_density_liquid");
+  rho_key_ = Keys::readKey(*plist_, domain_, "mass density liquid", "mass_density_liquid");
```

Comment:
just a suggestion. I think it would be nice to have dens_key_ for mass_density_liquid and molar_dens_key_ for molar_density_liquid

Reply:
I'm not sure what you mean.  This uses dens_key_ now, and "readKey" means it gets read from the parameter list (this call means you can use "molar density liquid key" or "molar density liquid key suffix" in the input spec, and they will result in "DOMAIN-KEY" pairs.  The "molar_density_liquid" here sets the default value -- you can override it.

Reply:
Oh, I see what you're saying, swapping the names. Hmm.

Reply:
Ok, I'm changing them to mass_dens_key_ and molar_dens_key_ to be consistent with Richards PK.

Reply:
yeah, I see my comment was not clear, but you got it now.